tests/ttnn/python_api_testing/non_working_unit_tests/grayskull/test_eltwise_pow_float.py

```diff
@@ -15,7 +15,7 @@
 def run_pow_tests(input_shape, dtype, dlayout, in_mem_config, output_mem_config, data_seed, device):
     torch.manual_seed(data_seed)
 
-    x = torch.Tensor(size=input_shape[0]).uniform_(-100, 100).to(torch.bfloat16)
+    x = torch.Tensor(size=input_shape[0]).uniform_(-100, 100)
```

Comment:
* should we move this test out of `non_working_unit_tests` folder? 
* previously we were testing bfloat16 pow operation, now it's pow operation for float32, right? 
If true, should we just update range for x (`...uniform_(-5, 5).to(torch.bfloat16)` for example) to be a better match for previous test intention?

Reply:
Moved the file.

Reply:
Even if we minimize the range again the values will be a larger number if it is bf16 in torch, then if we don't do conversion the result will be nan. In TT version of pwer of we are explicitly checking for negative value, if the value is negative it will return nan

Reply:
In TT we explicitly added a condition to return NaN if the values are negative. In torch for BF16 Nan is represented in large values instead of NaN which will be causing PCC drop hence removed the conversion part in torch ref code in this file